meta/main.yml

```diff
@@ -12,6 +12,7 @@ galaxy_info:
   # Kali Linux, so it makes sense to force the installation of Ansible
   # 2.10 or newer.
   min_ansible_version: 2.10
+  namespace: cisagov
```

Comment:
We should create a PR to add this in https://github.com/cisagov/skeleton-ansible-role.

Reply:
I did this a few weeks ago in cisagov/skeleton-ansible-role#85, which will be unleashed upon the next Kraken release.